tests/e2e/where_now_test.go

```diff
@@ -54,26 +55,43 @@ func TestWhereNowNoUUID(t *testing.T) {
 
 	pn := pubnub.NewPubNub(config)
 	pn.Subscribe().Channels([]string{"ch1"}).Execute()
-	time.Sleep(2 * time.Second)
+	time.Sleep(500 * time.Millisecond)
```

Comment:
Nice to see the tests spending less time. It will be nice in the future to remove these type of tests that require delays.

Reply:
Yes. I can't wait to have them fast and reliable 👍